templates/include/telemetry/environment.1.schema.json

```diff
@@ -509,7 +509,7 @@
               }
             },
             "monitors": {
-              "type": [ "array", "object" ],
+              "type": [ "array" ],
```

Comment:
```suggestion
              "type": "array",
```

Reply:
Good point. I did this in the GH editor, not a good decision for something that required a build.